src/main/java/de/tum/in/www1/artemis/service/TextSubmissionService.java

```diff
@@ -228,18 +228,14 @@ public TextSubmission save(TextSubmission textSubmission) {
      * @return a list of text submissions for the given exercise id
      */
     public List<TextSubmission> getTextSubmissionsByExerciseId(Long exerciseId, boolean submittedOnly, boolean examMode, int correctionRound) {
```

Comment:
It seems that the parameter `correctionRound` is not used any more, so we could remove it.

Reply:
You are right, it's not used anymore. I removed it.